packages/local_auth/local_auth_platform_interface/lib/types/biometric_type.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Various types of biometric authentication.
```

Comment:
This seems like a very good opportunity to add `strong` and `weak` per https://github.com/flutter/plugins/pull/3763#issuecomment-815033841

They won't be used yet, but it sets us up for a change we know we're going to make later anyway, without having to propagate it through the packages.

Reply:
Added